{
  "uid" : "98d329682ef9af0",
  "name" : "test_post_body_negative[connection_POST_negative_Invalid field types and values]",
  "fullName" : "api.test_request_body.test_post#test_post_body_negative",
  "historyId" : "479bea9a61ec3ee079b4453110258a53",
  "time" : {
    "start" : 1618295196394,
    "stop" : 1618295196394,
    "duration" : 0
  },
  "description" : "\n    Negative cases of request body testing\n    Includes sets of invalid field values - out of boundary values,\n    nullable and required if not possible, fields with incorrect types etc.\n    ",
  "descriptionHtml" : "<pre><code>Negative cases of request body testing\nIncludes sets of invalid field values - out of boundary values,\nnullable and required if not possible, fields with incorrect types etc.\n</code></pre>\n",
  "status" : "failed",
  "statusMessage" : "AssertionError: Expecting status code 201 but got 422",
  "statusTrace" : "request = <SubRequest 'prepare_post_body_data' for <Function test_post_body_negative[connection_POST_negative_Invalid field types and values]>>\napp_fs = <tests.utils.api_objects.APPApi object at 0x7f346058ce80>\n\n    @pytest.fixture()\n    def prepare_post_body_data(request, app_fs: APPApi):\n        \"\"\"\n        Fixture for preparing test data for POST request, depending on generated test datasets\n        \"\"\"\n        test_data_list: List[TestDataWithPreparedBody] = request.param\n>       valid_request_data = DbFiller(app=app_fs).generate_valid_request_data(\n            endpoint=test_data_list[0].test_data.request.endpoint, method=Methods.POST\n        )\n\ntests/api/test_request_body/test_post.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/test_data/db_filler.py:39: in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\ntests/test_data/db_filler.py:80: in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\ntests/test_data/db_filler.py:108: in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\ntests/test_data/db_filler.py:88: in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\ntests/utils/api_objects.py:72: in exec_request\n    status_code_should_be(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [422]>, status_code = <HTTPStatus.CREATED: 201>\n\n    @allure.step(\"Response status code should be equal {status_code}\")\n    def status_code_should_be(response: Response, status_code=HTTPStatus.OK):\n        \"\"\"Assert response status code\"\"\"\n>       assert (\n            response.status_code == status_code\n        ), f\"Expecting status code {status_code} but got {response.status_code}\"\nE       AssertionError: Expecting status code 201 but got 422\n\ntests/steps/asserts.py:16: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "cmd_opts",
    "time" : {
      "start" : 1618295160425,
      "stop" : 1618295160425,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "app_fs",
    "time" : {
      "start" : 1618295196395,
      "stop" : 1618295197817,
      "duration" : 1422
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Run container: local/app:smBLoPUSna",
      "time" : {
        "start" : 1618295196411,
        "stop" : 1618295196770,
        "duration" : 359
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "prepare_post_body_data",
    "time" : {
      "start" : 1618295197818,
      "stop" : 1618295197932,
      "duration" : 114
    },
    "status" : "failed",
    "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/api/test_request_body/test_post.py\", line 32, in prepare_post_body_data\n    valid_request_data = DbFiller(app=app_fs).generate_valid_request_data(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 39, in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 80, in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 108, in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 88, in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
    "steps" : [ {
      "name" : "Generate valid request data",
      "time" : {
        "start" : 1618295197818,
        "stop" : 1618295197932,
        "duration" : 114
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 39, in generate_valid_request_data\n    \"data\": self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 80, in _get_or_create_data_for_endpoint\n    data = self._prepare_data_for_object_creation(endpoint=endpoint, force=force)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 108, in _prepare_data_for_object_creation\n    fk_data = self._get_or_create_data_for_endpoint(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/test_data/db_filler.py\", line 88, in _get_or_create_data_for_endpoint\n    response = self.app.exec_request(\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
      "steps" : [ {
        "name" : "Send LIST /cluster/",
        "time" : {
          "start" : 1618295197818,
          "stop" : 1618295197876,
          "duration" : 58
        },
        "status" : "passed",
        "steps" : [ {
          "name" : "Response status code should be equal <HTTPStatus.OK: 200>",
          "time" : {
            "start" : 1618295197876,
            "stop" : 1618295197876,
            "duration" : 0
          },
          "status" : "passed",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "response",
            "value" : "<Response [200]>"
          }, {
            "name" : "status_code",
            "value" : "<HTTPStatus.OK: 200>"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : false,
          "hasContent" : true
        } ],
        "attachments" : [ {
          "uid" : "bb474dd50166054a",
          "name" : "Full request log",
          "source" : "bb474dd50166054a.txt",
          "type" : "text/plain",
          "size" : 504
        } ],
        "parameters" : [ ],
        "stepsCount" : 1,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      }, {
        "name" : "Send POST /cluster/",
        "time" : {
          "start" : 1618295197876,
          "stop" : 1618295197932,
          "duration" : 56
        },
        "status" : "failed",
        "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
        "statusTrace" : "  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/utils/api_objects.py\", line 72, in exec_request\n    status_code_should_be(\n  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
        "steps" : [ {
          "name" : "Response status code should be equal <HTTPStatus.CREATED: 201>",
          "time" : {
            "start" : 1618295197932,
            "stop" : 1618295197932,
            "duration" : 0
          },
          "status" : "failed",
          "statusMessage" : "AssertionError: Expecting status code 201 but got 422\n",
          "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/truly_automated_tests/truly_automated_tests/tests/steps/asserts.py\", line 16, in status_code_should_be\n    assert (\n",
          "steps" : [ ],
          "attachments" : [ ],
          "parameters" : [ {
            "name" : "response",
            "value" : "<Response [422]>"
          }, {
            "name" : "status_code",
            "value" : "<HTTPStatus.CREATED: 201>"
          } ],
          "stepsCount" : 0,
          "attachmentsCount" : 0,
          "shouldDisplayMessage" : true,
          "hasContent" : true
        } ],
        "attachments" : [ {
          "uid" : "4b8f72d0c8c337b2",
          "name" : "Full request log",
          "source" : "4b8f72d0c8c337b2.txt",
          "type" : "text/plain",
          "size" : 969
        } ],
        "parameters" : [ ],
        "stepsCount" : 1,
        "attachmentsCount" : 1,
        "shouldDisplayMessage" : false,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "endpoint",
        "value" : "<Endpoints.Connection: Endpoint(path='connection', methods=[<Methods.GET: Method(function=<function get at 0x7f3460942a60>, url_template='/{name}/{id}/', default_success_code=<HTTPStatus.OK: 200>)>, <Methods.LIST: Method(function=<function get at 0x7f3460942a60>, url_template='/{name}/', default_success_code=<HTTPStatus.OK: 200>)>, <Methods.POST: Method(function=<function post at 0x7f3460942c10>, url_template='/{name}/', default_success_code=<HTTPStatus.CREATED: 201>)>], data_class=<class 'tests.utils.data_classes.ConnectionFields'>)>"
      }, {
        "name" : "method",
        "value" : "<Methods.POST: Method(function=<function post at 0x7f3460942c10>, url_template='/{name}/', default_success_code=<HTTPStatus.CREATED: 201>)>"
      } ],
      "stepsCount" : 4,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 5,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "image",
    "time" : {
      "start" : 1618295160425,
      "stop" : 1618295164299,
      "duration" : 3874
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Run container: app:refs_pull_6_merge",
      "time" : {
        "start" : 1618295160461,
        "stop" : 1618295161098,
        "duration" : 637
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "app_fs::fin",
    "time" : {
      "start" : 1618295197954,
      "stop" : 1618295198299,
      "duration" : 345
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Gather /var/www/html/storage/app/ from APP container: c028d827f30330c010920ff0277cba01edd13a6996345fc93043e16f2dbc4f1c",
      "time" : {
        "start" : 1618295197954,
        "stop" : 1618295198070,
        "duration" : 116
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "7a904680e507fe9c",
        "name" : "test_post_body_negative[connection_POST_negative_Invalid field types and values]_1618295197.9543285.tgz",
        "source" : "7a904680e507fe9c.tgz",
        "type" : "application/gzip",
        "size" : 1102
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "image::fin",
    "time" : {
      "start" : 1618295200177,
      "stop" : 1618295200189,
      "duration" : 12
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Body tests"
  }, {
    "name" : "subSuite",
    "value" : "POST"
  }, {
    "name" : "parentSuite",
    "value" : "API"
  }, {
    "name" : "tag",
    "value" : "negative"
  }, {
    "name" : "host",
    "value" : "fv-az178-863"
  }, {
    "name" : "thread",
    "value" : "5034-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "api.test_request_body.test_post"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "prepare_post_body_data",
    "value" : "[TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da430, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da550, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da640, test_body={'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da730, test_body={'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da7f0, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da910, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606daa00, test_body={'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606daaf0, test_body={'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)})]"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "negative" ]
  },
  "source" : "98d329682ef9af0.json",
  "parameterValues" : [ "[TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da430, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False), 'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da550, test_body={'filesystem_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da640, test_body={'cluster_id': PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[..., PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False)]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da730, test_body={'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da7f0, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False), 'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606da910, test_body={'filesystem_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected filesystem id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606daa00, test_body={'cluster_id': PreparedFieldValue(value=2147483648, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=ForeignKey(_sp_vals_positive=None, _sp_vals_negative=[PreparedFieldValue(value=100, generated_value=False, error_messages=['The selected cluster id is invalid.'], f_type=..., drop_key=False), ...]), drop_key=False)}), TestDataWithPreparedBody(test_data=POST connection and expect 422 status code. at 0x7f34606daaf0, test_body={'name': PreparedFieldValue(value='nYuSywUkZlktOzXgXLsDPztSdzmWHEDUdORFRVSllyAiQdjgXzpYgNmGuiIJciQmrewSUbMhNtYBDXYXSnHtZsVCPyoKYqxvJiYNsVHHTyaoIIIdVBvJntcSDIofJbsBlwxUDpiWNBPPIFgZveAsZdQFXOtqrzfgZPxDKCMGIYkZWxGfZdSEjuzNJAnzMMxzkAXXtwsXZIwWHdjYyOmWqTkVrMvwTtfwbYwFExVHZFyelmmWuCWBhWTqYYyzlKUQ', generated_value=False, error_messages=['The name may not be greater than 255 characters.'], f_type=String(_sp_vals_positive=['s', '!@#$%^&*\\\\/{}[]', 'UrwCQDGOxbKoIofEhAVIyYGPvVAIWLntTcZfudIOJrJSigugXvekfdtfHxzLYEONeLICoPWylQEBZfcdRtIkkYGOvTbJjeYiFnmsjSDgSjnpgNZlZXrVvAvzGRzweJxoqbWrGejbivoQkzgwhIWWZzYsQfBCINaAXTtIFQrIJHdBbOVsMfhGsFivKZBLMkZoGuzgIlFbpYjgcdjooCmLoGjctIZAfBqLuCqAzPhFMpWemwWBGzSgwOLfEasazWs'], _sp_vals_negative=[...]), drop_key=False)})]" ]
}